code/modules/tgui/states/zlevel_state.dm

```diff
@@ -4,13 +4,11 @@
  * Only checks that the Z-level of the user and src_object are the same.
  */
 
-/*
 GLOBAL_DATUM_INIT(z_state, /datum/ui_state/z_state, new)
 
 /datum/ui_state/z_state/can_use_topic(src_object, mob/user)
 	var/turf/turf_obj = get_turf(src_object)
 	var/turf/turf_usr = get_turf(user)
 	if(turf_obj && turf_usr && turf_obj.z == turf_usr.z)
-		return STATUS_INTERACTIVE
-	return STATUS_CLOSE
-*/
```

Comment:
I suppose it depends on if we want it in future or not, the whole file should be deleted instead of commenting it out if we are dropping it.
I'll delete it for now